recipes/twitch-native-ipc/all/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class TwitchNativeIpcTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self):
+            #bin_path = os.path.join("bin", "example")
+            cmd = "{} && {}".format(os.path.join("bin", "example"), os.path.join("bin", "example2"))
+            self.run(cmd, run_environment=True)
```

Comment:
```suggestion
            self.run(os.path.join("bin", "example"), run_environment=True)
            self.run(os.path.join("bin", "example2"), run_environment=True)
```

Run them separated, any error can be detected easier.